test/coexistence/README.md

```diff
@@ -0,0 +1,33 @@
+# Tanzu CLI Coexistence Testing
+
+## Overview
+
+The purpose of this testing is to test the unification of Tanzu CLI to be able to use both new and legacy Tanzu CLI versions.
+
+Tanzu CLI Coexistence testing is to ensure that both legacy Tanzu CLI and new Tanzu CLI can interoperate to know more about this go to [Using Legacy Tanzu CLI along with new Tanzu CLI](https://github.com/vmware-tanzu/tanzu-cli/blob/main/docs/quickstart/quickstart.md#using-the-legacy-version-of-the-cli-alongside-the-new-tanzu-cli)
```

Comment:
Suggestion, update the text as:
Tanzu CLI Coexistence testing ensures that both the legacy Tanzu CLI and the new Tanzu CLI can work together. To learn more about this, visit to [Using Legacy Tanzu CLI along with new Tanzu CLI](https://github.com/vmware-tanzu/tanzu-cli/blob/main/docs/quickstart/quickstart.md#using-the-legacy-version-of-the-cli-alongside-the-new-tanzu-cli)